### DISCUSSION TITLE
Omnisharp .net8

### DISCUSSION THREAD

**Discussion by lucasjesus682** [balta-io/forum#486] [Tira Dúvidas] [answered]
Oi, estou com uma dúvida que está me atrapalhando a progredir, estou acompanhando o curso de fundamentos em c# e devido ao novo template não entendi como funciona para configurar o Modern net, se alguém puder me dar uma forcinha :P.

ao final, queria que meu vscode aparecesse aquelas telinhas de quando alguma coisa está errada

![imagem_2024-02-04_164608052](https://github.com/balta-io/forum/assets/98325931/07bb47fb-6f71-4dd2-8919-7859514c7112)

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @lucasjesus682, como vai?

Qual SO está usando?

**Reply by lucasjesus682** [reply]
Estou usando o Windows 11 Home Single Language

**Reply by andrebaltieri** [reply]
Estranho, me parece que deu algum problema na instalação do seu .NET/VS Code.
Você consegue remover os dois e instalar novamente?
